src/features/SwapAssets/SwapAssetsFlow.tsx

```diff
@@ -212,12 +214,12 @@ const SwapAssetsFlow = (props: RouteComponentProps) => {
   }, [canYield]);
 
   const mounted = usePromise();
-  useEffectOnce(() => {
+  useEffect(() => {
     (async () => {
       const [fetchedAssets, fetchedFromAsset, fetchedToAsset] = await mounted(fetchSwapAssets());
       setSwapAssets(fetchedAssets, fetchedFromAsset, fetchedToAsset);
     })();
```

Comment:
This will cause unhandled promise rejections if something inside throws.

Reply:
What would you prefer for this?